### ISSUE TITLE
[Bug] Accessibility Show Focus Traps or Out-Of-Order Reading 

### ISSUE THREAD

**Issue by ahodder-charter** [xamarin/Xamarin.Forms#9653]
### Description
For both Android and iOS, we are experiencing multiple types of accessibility issues. Issues include out-of-order reading in which a view will be skipped and read after view later in the hierarchy. In iOS in particular, we are seeing focus traps in which voice over navigation cannot advance passed or before a collection of views. We have provided a sample project to demonstrate this as well as videos showing the issues we are seeing. 

### Steps to Reproduce

1. Create a new forms project
2. Create a static layout normally containing two orthogonal stack layouts (although any view group will work)
3. Navigating through the view hierarchy will trigger either a focus trap or view skipping

### Expected Behavior
Voiceover should be able to navigate views in a sane linear method. For us this is top to bottom, left to right.

### Actual Behavior
Views are either skipped or navigation gets stuck in a sub view group and voiceover is incapable of leaving the view without resorting to untypical navigation methods.

### Basic Information

- Version with issue: 4.5.0.142-pre as well as 4.4
- Last known good version: Unknown
- IDE: Rider and Visual Studio Mac
- Platform Target Frameworks: Android, iOS
  - iOS:  11.3
  - Android: 9 and 10
- Affected Devices:

### Screenshots

A test video is attached in to the sample project

### Workaround

None known.

[AccessibilityTestForms.zip](https://github.com/xamarin/Xamarin.Forms/files/4232086/AccessibilityTestForms.zip)

**Comment by conradh** [comment]
Issue confirmed. Can be related to upgrading to 4.x and upwards for us i assume.

**Comment by ahodder-charter** [comment]
What's the progress on this? Is there anything we can do to assist?
